### PR TITLE
Design fixes for Documentation page 

### PR DESCRIPTION
**Pull request by anishaswain**
Fixes:
- On hovers, need to turn to the light teal colour (#1BC1C9), not the dark 
- Several instances of Overpass font converted to Work Sans
- There was too much space around the cards (they were not left-aligned with the rest of the page content), and the three cards do NOT need to be sitting on an additional shadow card.
- The cards should all auto-adjust to be the height of the tallest card, making them all the same height 
- The three recent meetings cards were too far apart, so switched the spacing to `margin: var( --pf-global--spacer--md);`
- The `quicklinks` should be left-aligned. 
- The dropdowns for the cards (Pbench agent, Pbench server, and Dashboard) should NOT use all caps -- use sentence case.
- There was a bit of an issue with the interaction of the dropdown on the cards as well. When the card is expanded, the icon would be down, and it needs to swap to pointing to the right on closing.

### CODE REVIEW

**Review by npalaska — APPROVED** [review]

File: index.css [hunk 1]
```diff
@@ -241,4 +245,8 @@ button.pf-c-button.pf-m-link{
 
 .addPadding1, .addPadding2, .addPadding3{
   padding: var( --pf-global--spacer--xl );
-}
\ No newline at end of file
+}
+
+.hidden{
+  display: none;
+}
```
Comment (ndokos) [inline index.css:252]: No newline at end.

File: index.html [hunk 2]
```diff
@@ -90,8 +90,8 @@
                               </p>
                            </div>
                            <div class="landingImg">
-                              <div class="pf-c-card__header pf-c-title pf-m-md landingImageDiv div1">
-                                 <p class="cardHeader"><b>Placeholder Image</b></p>
+                              <div class="pf-c-card__header pf-c-title pf-m-md landingImageDiv">
+
```
Comment (ndokos) [inline index.html:94]: Ah, I can see this one ;-)

File: js/drawer.js [hunk 3]
```diff
@@ -2,28 +2,40 @@ $(document).ready(function(){
...
     });
 
     $("#dashboardBtn").click(function(){
         $("#dashboardToggle").slideToggle("flast","linear");  
         $('#dashboardBox').toggleClass("addPadding3");
+        $('#thirdGrid').toggleClass("maxHeight");
+        $('#dashboardOpenIcon').toggleClass("hidden");
+        $('#dashboardCloseIcon').toggleClass("hidden");
     });
 });
```
Comment (ndokos) [inline js/drawer.js:41]: No newline at end.

**Review by ndokos — APPROVED** [review]
A couple of minor preexisting nits (no newlines at end of a couple of files). Otherwise, whatever changes I was able to see looked good, but there are a bunch of changes that I wasn't able to verify visually (primarily because I don't know where to look for them).

**Review by portante — APPROVED** [review]

**Pull request merged**